### ISSUE TITLE
Raspberry Pi 4 8GB modified Dockerfile

### ISSUE THREAD

**Issue by sincze** [traccar/traccar-docker#61]
Current dockerfile did for some reason not work on my Pi 4 so modified it a bit.

```
FROM debian:latest

ENV TRACCAR_VERSION 4.10

WORKDIR /opt/traccar

RUN set -ex && \
    apt-get update &&\
    TERM=xterm DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends unzip wget default-jre && \
    wget -qO /tmp/traccar.zip https://github.com/traccar/traccar/releases/download/v$TRACCAR_VERSION/traccar-other-$TRACCAR_VERSION.zip && \
    unzip -qo /tmp/traccar.zip -d /opt/traccar && \
    apt-get autoremove --yes unzip wget && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/* /tmp/*

ENTRYPOINT ["java", "-Xms512m", "-Xmx512m", "-Djava.net.preferIPv4Stack=true"]

CMD ["-jar", "tracker-server.jar", "conf/traccar.xml"]
```

To create the Docker:
```
sudo nano Dockerfile (insert text above)
docker build -t debian-traccar-nginx .

```
After creating a volume **traccar_raspberry** under docker volumes 

```
docker run -d \
  --name Traccar \
  -p 5002:5002/tcp \
  -p 5002:5002/udp \
  -p 5055:5055/tcp \
  -p 5093:5093/tcp \
  -p 5093:5093/udp \
  -p 82:8082 \
  --mount source=traccar_raspberry,target=/opt/traccar \
  --restart=unless-stopped \
  debian-traccar-nginx:latest 
```

I am now a happy man!

**Comment by jamu85** [comment]
Causes 
$ docker logs 6f771fbbd26a
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/opt/traccar/lib/guice-4.2.3.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

on my rpi4 8gb

**Comment by tananaev** [comment]
It's just a warning, not an error.

**Comment by magna-z** [comment]
Multi-platform docker images released for current version 4.11. [Details in README.md](https://github.com/traccar/traccar-docker#available-multi-platform-images)

**Comment by ippocratis** [comment]
> Multi-platform docker images released for current version 4.11. [Details in README.md](https://github.com/traccar/traccar-docker#available-multi-platform-images)

Run on a rpi400 the Debian arm64 image and threw Java exec format errors
this modified Dockerfile worked
